src/games/index.js

```diff
@@ -1,9 +1,25 @@
-const Test1 = require('./test-1');
-const Test2 = require('./test-2');
-const Draw = require('./draw');
+const Test1 = require("./test-1");
+const Test2 = require("./test-2");
+const Draw = require("./draw");
+const MoveTest = require("./move-test");
+const Slaps = require("./slaps");
+const LayerTest = require("./layer-test");
+const NameTest = require("./name-test");
+const SpriteTest = require("./sprite-test");
+const TextTest = require("./text-test");
+const WordMatch = require("./word-match");
+const HomeButtonTest = require("./home-button-test");
 
 module.exports = {
+    HomeButtonTest,
     Draw,
+    LayerTest,
+    MoveTest,
+    NameTest,
+    Slaps,
+    SpriteTest,
     Test1,
-    Test2
+    Test2,
+    TextTest,
+    WordMatch
```

Comment:
all of these games currently run off of the screen since we never start a new row. need to fix that